utils/utils.py

```diff
@@ -1,12 +1,37 @@
 """Utility for json conversion and selenium setup"""
 
+from dataclasses import dataclass
 import datetime
 import json
 from selenium import webdriver
 from flask_mongoengine import BaseQuerySet
 from pymongo.command_cursor import CommandCursor
 
 
+@dataclass
+class ScrapingTask:
+    """Dataclass for Scraping Task Status"""
+
+    quote: str = "NOT_STARTED"
+    author: str = "NOT_STARTED"
```

Comment:
Use Enum values here.
```suggestion
    quote: str = TaskStatus.NOT_STARTED.value
    author: str = TaskStatus.NOT_STARTED.value
```